marklogic-data-hub/src/test/java/com/marklogic/hub/flow/TemporalWriteTest.java

```diff
@@ -68,15 +76,44 @@ void writeTemporalFile() {
             "declareUpdate();\n" +
             "temporal.documentDelete('temporal/test', '/temporal/ingestion/test.json');";
 
-        try{
+        try {
             runAsDataHubOperator().newStagingClient().newServerEval().javascript(deleteTemporalDoc).eval();
-        }
-        catch (Exception e){
+        } catch (Exception e) {
             logger.error("Document deletion failed: ", e);
             Assertions.fail("dh-operator should be able to delete the document");
         }
     }
 
+    @Test
+    void ingestDocWithTemporalCollecion() {
+        String collections = "fruits,stuff,temporal/test";
+        ObjectNode workUnit = objectMapper.createObjectNode();
+        workUnit.put("uriprefix", "/bulkJavaTest/");
+        workUnit.put("collections", collections);
+        runAsDataHubOperator();
+        InputEndpoint endpoint = InputEndpoint.on(
+            getHubClient().getStagingClient(),
+            stagingModulesClient.newTextDocumentManager().read("/data-hub/5/data-services/ingestion/bulkIngester.api", new StringHandle())
+        );
+        bulkInputCaller = endpoint.bulkCaller();
+        bulkInputCaller.setEndpointState("{}".getBytes());
+        bulkInputCaller.setWorkUnit(new JacksonHandle(workUnit));
+
+        Stream<InputStream> input = Stream.of(
+            asInputStream("{\"fruitName\":\"pineapple\", \"fruitColor\":\"green\"}")
+        );
+        input.forEach(bulkInputCaller::accept);
+        bulkInputCaller.awaitCompletion();
+
+        DocumentMetadataHandle metadata = getDocumentMetadata();
+        assertEquals(collections.split(",").length + 2, metadata.getCollections().size());
+        assertTrue(metadata.getCollections().contains("fruits"));
+        assertTrue(metadata.getCollections().contains("stuff"));
+        assertTrue(metadata.getCollections().contains("temporal/test"));
+        assertTrue(metadata.getCollections().contains("latest"));
```

Comment:
This should have an assertion stating that because it's in the "latest" collection, that means that "temporal/test" was recognized as a temporal collection. Otherwise there's no indication here that the document was inserted via temporal.documentInsert.